UsingRTT.md

```diff
@@ -167,8 +165,8 @@ target extended-remote \\.\COM10
 
 Target RTT output will appear in the terminal, and what you type in the terminal will be sent to the RTT input of the target.
 
-### linux
-On linux, install [udev rules](https://github.com/blacksphere/blackmagic/blob/master/driver/99-blackmagic.rules). Disconnect and re-connect the BMP. Check the device shows up in /dev/ :
+### Linux
+On Linux, install [udev rules](https://github.com/blacksphere/blackmagic/blob/master/driver/99-blackmagic.rules). Disconnect and re-connect the BMP. Check the device shows up in /dev/ :
```

Comment:
The link here needs updating both because this points to the old repo location and is a deadlink to the udev rules files.

We'd suggest two links, one for "distros that use UUCP (Arch/Manjaro)" and one for "distros that use plugdev (Debian/Ubuntu)", seeing this is why there are two .rules files.

Reply:
ohh DUH... :facepalm:  good catch

Reply:
To be fair we should not describe this here at all... it should link to the global documentation instead.